src/main/java/org/ow2/proactive/connector/iaas/rest/NodeCandidateRest.java

```diff
@@ -47,18 +46,24 @@
     @Autowired
     public NodeCandidateService nodeCandidateService;
 
-    @POST
+    @GET
```

Comment:
I thought we might have problem to use HTTP GET method if we want to consume JSON as request body. Here is a ref: https://stackoverflow.com/questions/978061/http-get-with-request-body